packages/core/package.json

```diff
@@ -31,6 +31,7 @@
   },
   "homepage": "https://strudel.tidalcycles.org",
   "dependencies": {
+    "@tauri-apps/api": "^1.4.0",
```

Comment:
i think we should try to not have that dependency in the core module to keep it properly encapsulated